### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Zmoki**

### CODE REVIEW

**Comment by Zmoki** [comment]
[Widget v3: Обновить readme в гитхабе](https://trello.com/c/TPGwIdPJ/130-widget-v3-readme)

File: README.markdown [hunk 1]
```diff
@@ -1,144 +1,167 @@
...
-## Using with Bower
+Default exported bundle for npm and other packages managers is `uploadcare.full.min.js`.
 
-Install uploadcare using `bower` command:
+## Install
 
-```sh
-$ bower install uploadcare
-```
+You’re free to choose from the install methods listed below.
```
Comment (dayton1987) [inline README.markdown]: E.g. You're free to pick an install method from the list below:

File: README.markdown [hunk 2]
```diff
@@ -1,144 +1,167 @@
...
-    bundle exec rails server
-    
-Open http://127.0.0.1:3000/. Follow any link. 
-There's going to be a widget or three. Edit code and reload page :-)
+See [full demo](https://github.com/uploadcare/uploadcare-widget-react-demo/).
 
+### Usage with Angular
 
-### Testing
+We have Angular 1 directive for Uploadcare Widget.
```
Comment (dayton1987) [inline README.markdown]: `a | the` directive

File: README.markdown [hunk 3]
```diff
@@ -1,144 +1,167 @@
...
+See [angular-uploadcare](https://github.com/uploadcare/angular-uploadcare).
 
-[Jasminerice](https://github.com/bradphelan/jasminerice) 
-installed under the `test/dummy/` Rails app.
+For Angular 2 maybe [this demo](https://plnkr.co/edit/6caWQ6cct4L3715LehxZ?p=preview) can be useful.
```
Comment (dayton1987) [inline README.markdown]: For Angular 2, this demo `could | might` be useful.

File: README.markdown [hunk 4]
```diff
@@ -1,144 +1,167 @@
...
 
-To run tests in your browser go to http://127.0.0.1:3000/jasmine.
+## Configuration
 
-For more information see 
-[jasminerice docs](https://github.com/bradphelan/jasminerice).
+The widget is highly customizable through widget options.
+All configuration options and ways to set options
```
Comment (dayton1987) [inline README.markdown]: All configuration options `together | along` with ways to set them are described in `these | the | our` docs.

File: README.markdown [hunk 5]
```diff
@@ -1,144 +1,167 @@
...
+You might not want to use all the features that
+[our widget](https://uploadcare.com/documentation/widget/) exhibits.
+Or, perhaps, you might want to redesign the user experience
+without having to reinvent the wheel.
+Maybe, you're in pursuit of building a UI on top of the widget.
+For all of those use cases, we provide a
+[JavaScript API](https://uploadcare.com/documentation/javascript_api/).
+Feel free to control the default widget with it,
+or make use of its standalone components that
+can be combined with your own solutions.
```
Comment (dayton1987) [inline README.markdown]: line 145-154 надо бы переписать по-хорошему, но это не критично

File: README.markdown (hunk 1 above)
Comment (Zmoki) [inline README.markdown]: I like `You’re free to choose from the install methods listed below.`

**Comment by Zmoki** [comment]
@dayton1987 bump
https://github.com/uploadcare/uploadcare-widget/tree/new-github-readme#readme

File: README.md [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+
+## Install
+
+You’re free to choose from the install methods listed below.
+
+### CDN
+
+Embed our client library via `<script>` tag in the `<head>`
+section of each page where you’d like to make use of Uploadcare Widget.
+Just use this CDN link to use the widget version with built-in jQuery,
```
Comment (dayton1987) [inline README.md]: 2 x `use` in a single sentence. `to embed the widget version`?

File: README.md [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+You might not want to use all the features that
+[our widget](https://uploadcare.com/documentation/widget/) exhibits.
+Or, perhaps, you might want to redesign the user experience
+without having to reinvent the wheel.
+Maybe, you're in pursuit of building a UI on top of the widget.
+For all of those use cases, we provide a
+[JavaScript API](https://uploadcare.com/documentation/javascript_api/).
+Feel free to control the default widget with it,
+or make use of its standalone components that
+can be combined with your own solutions.
```
Comment (dayton1987) [inline README.md]: I think I've already mentioned I don't like this paragraph, but we can still fix this later.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,220 @@
...
+        <td>Opera Mini: Last</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Development
+
+Guide how to development Uploadcare Widget
+at [DEVELOPMENT.md](https://github.com/uploadcare/uploadcare-widget/blob/master/DEVELOPMENT.md).
```
Comment (dayton1987) [inline README.md]: Yowza, `Check out the Uploadcare Widget development guide [here](<link>)`

File: DEVELOPMENT.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
+# Development Uploadcare Widget
```
Comment (dayton1987) [inline DEVELOPMENT.md]: Uploadcare Widget development guide?

File: DEVELOPMENT.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+
+Clone the repository.
```
Comment (dayton1987) [inline DEVELOPMENT.md]: Clone the repository. Now.

`Make sure you're making changes in your local copy of our repo.`?

File: DEVELOPMENT.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+
+### Enviroment
+
+You need a working Ruby 2.0.0 and above environment with [Bundler](http://bundler.io).
+
+#### Vagrant
+
+If you don't want to raise the environment on your machine, you can use [Vagrant](https://www.vagrantup.com/).
```
Comment (dayton1987) [inline DEVELOPMENT.md]: raise the environment?

File: DEVELOPMENT.md [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+Just [install Vagrant](https://www.vagrantup.com/docs/installation/) and [VirtualBox](https://www.virtualbox.org/) or [other provider](https://www.vagrantup.com/docs/getting-started/providers.html).
+
```
Comment (dayton1987) [inline DEVELOPMENT.md]: other `<virtual environment>` provider?

File: DEVELOPMENT.md [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+After open command line, go to this folder and run:
```
Comment (dayton1987) [inline DEVELOPMENT.md]: Then, run the following via you console: ?

File: DEVELOPMENT.md [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+
+    vagrant up
+    vagrant ssh
+
+#### Local environment (without Vagrant)
+
+Inside folder run
```
Comment (dayton1987) [inline DEVELOPMENT.md]: `Use your command line to run the following from your local repo`?

File: DEVELOPMENT.md [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+    bundle install
+    cd ./test/dummy
+    bundle install
+
+### Build
+
+* `bundle exec rake js:latest:build` to build assets
+  to the **pkg/latest** directory (with the “latest” suffix).
+* `bundle exec rake js:release:build` to build assets
+  to the **pkg/version** folder (with the current version suffix).
```
Comment (dayton1987) [inline DEVELOPMENT.md]: folder >> directory

File: DEVELOPMENT.md [hunk 16]
```diff
@@ -0,0 +1,70 @@
...
+  The version is specified in `lib/uploadcare-widget/version.rb`.
```
Comment (dayton1987) [inline DEVELOPMENT.md]: Which version? Feels like clarifying

File: DEVELOPMENT.md [hunk 17]
```diff
@@ -0,0 +1,70 @@
...
+
+### Development
+
+Go to `test/dummy/`. There is a simple Rails app. Run it:
+
+    bundle exec rails server
+    
+Open http://127.0.0.1:3000/. Follow any link.
```
Comment (dayton1987) [inline DEVELOPMENT.md]: `http://<>` ?

File: DEVELOPMENT.md [hunk 18]
```diff
@@ -0,0 +1,70 @@
...
+  The version is specified in `lib/uploadcare-widget/version.rb`.
+
+### Development
+
+Go to `test/dummy/`. There is a simple Rails app. Run it:
+
+    bundle exec rails server
+    
+Open http://127.0.0.1:3000/. Follow any link. 
+There's going to be a widget or three. Edit code and reload page :-)
```
Comment (dayton1987) [inline DEVELOPMENT.md]: `reload the page`

File: DEVELOPMENT.md [hunk 19]
```diff
@@ -0,0 +1,70 @@
...
+
+
+### Testing
+
+[Jasminerice](https://github.com/bradphelan/jasminerice)
```
Comment (dayton1987) [inline DEVELOPMENT.md]: For testing purposes, use ... ?

File: DEVELOPMENT.md [hunk 20]
```diff
@@ -0,0 +1,70 @@
...
+Open http://127.0.0.1:3000/. Follow any link. 
+There's going to be a widget or three. Edit code and reload page :-)
+
+
+### Testing
+
+[Jasminerice](https://github.com/bradphelan/jasminerice) 
+installed under the `test/dummy/` Rails app.
+
+To run tests in your browser go to http://127.0.0.1:3000/jasmine.
```
Comment (dayton1987) [inline DEVELOPMENT.md]: `browser, ` `http://<>` ?

File: DEVELOPMENT.md [hunk 21]
```diff
@@ -0,0 +1,70 @@
...
+
+For more information see
```
Comment (dayton1987) [inline DEVELOPMENT.md]: For more information, see

File: DEVELOPMENT.md [hunk 22]
```diff
@@ -0,0 +1,70 @@
...
+installed under the `test/dummy/` Rails app.
+
+To run tests in your browser go to http://127.0.0.1:3000/jasmine.
+
+For more information see 
+[jasminerice docs](https://github.com/bradphelan/jasminerice).
+
+#### guard-jasmine
+
+To run tests in a terminal you must first
```
Comment (dayton1987) [inline DEVELOPMENT.md]: In order to run tests in a terminal, you should first

File: DEVELOPMENT.md [hunk 23]
```diff
@@ -0,0 +1,70 @@
...
+
+For more information see 
+[jasminerice docs](https://github.com/bradphelan/jasminerice).
+
+#### guard-jasmine
+
+To run tests in a terminal you must first 
+[install phantomjs](https://github.com/guard/guard-jasmine#phantomjs).
+
+Then you have two options:
```
Comment (dayton1987) [inline DEVELOPMENT.md]: `Then you have two options: your son dies or you send us the money, biatch`

Then, there are two options:

**Pull request merged**